content/influxdb/v2.0/reference/syntax/annotated-csv/extended.md

```diff
@@ -325,6 +326,18 @@ Use the `±HHmm` format to specify the timezone offset relative to UTC.
 #timezone -0600
 ```
 
+### concat
+
+The `#concat` annotation adds a new column that is concatenated from existing columns according to bash-like string interpolation literal with variables referencing existing column labels.
+
+For example:
+
+```
+#concat,string,fullName,${firstName} ${lastName}
+```
+
+This is especially useful when constructing a timestamp from multiple columns.
```

Comment:
Shall we add an example similar to the one that Russ demo'd? Or example here: https://github.com/influxdata/influxdb/issues/18741